### ISSUE TITLE
JFrog xray scan fail

### ISSUE THREAD

**Issue by ryanhamilton** [timestored/pulseui#170]
user-ag reports "xray scan fail". Seems likely to be jfrog.
Seen similar issues with bduck user-rd but resolved them by knowing what dependencies to update.

**Comment by ryanhamilton** [comment]
user-rd - Similar issues.

![image](https://github.com/timestored/pulseui/assets/7273208/43a0c9a6-508d-4c7a-83af-9f129ef42e9b)

**Comment by ryanhamilton** [comment]
- Large number of dependencies updated.  
- OWASP scanner passes.  (https://owasp.org/www-community/Vulnerability_Scanning_Tools)
- Black duck detects wrong code versions = fails

Black duck synopsys which it turns out, shows false positives for common java dependencies:
https://community.synopsys.com/s/article/General-guidance-about-multiple-false-positives-from-Black-Duck-scan
" Scenario 2: If there are components that come from large frameworks (such as Java, OpenSSL, etc) which are commonly impacted by false positives (due primarily to ambiguous matching)."
If any folder looks similar to a previous version, it detects all versions as being installed. So there's no sensible way to get it to pass.
How much hassle or time does it take you to get an xray scan done?
If it's reasonable, we could aim for Friday.

**Comment by ryanhamilton** [comment]
```
$ gradle dependencyCheckAnalyze
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

> Task :server:dependencyCheckAnalyze
Verifying dependencies for project server
Checking for updates and analyzing dependencies for vulnerabilities
Generating report for project server
Found 0 vulnerabilities in project server

Region [NODEAUDIT] : Not alive and dispose was called, filename: NODEAUDIT
Region [CENTRAL] : Not alive and dispose was called, filename: CENTRAL
Region [POM] : Not alive and dispose was called, filename: POM

BUILD SUCCESSFUL in 50s
1 actionable task: 1 executed
```

**Comment by ryanhamilton** [comment]
JFrog I setup a maven repository, add the jar and it detects no BoM:

![image](https://github.com/timestored/pulseui/assets/7273208/5e5fa92c-89f4-4fcf-ae31-4b0f3c8f3550)

**Comment by ryanhamilton** [comment]
Released in 2.20.
